apps/cli-gui/src/styles.scss

```diff
@@ -1,5 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
 @import './assets/material';
+@import './assets/core/reset';
```

Comment:
sass files shouldn't be in assets folder. Move them to ׳src/styles׳ or somewhere else.
Also move material.scss (validate this with @Danieliverant)

Reply:
@ronnetzer right, let this PR pass like this I will move them tomorrow

Reply:
Why? I prefer to resolve it before. Don't see any reason not to solve it right now

Reply:
if @eladzipper96 doesn't mind just moving both of them to "styles" fine by me, I just don't want somebody else fixing my mistakes :)

Reply:
sure ill do it, moving to src/styles